EIPS/eip-4750.md

```diff
@@ -127,7 +141,7 @@ In addition to container format validation rules above, we extend code section v
 
 ### `RETF` in the top frame ends execution vs exceptionally halts
 
-Alternative logic for executing `RETF` in the top frame could be to exceptionally halt execution, because there is arguably no caller for the starting function. This would mean that return stack is initialized as empty, and `RETF` exceptionally aborts when return stack is empty.
+execution, because there is arguably no caller for the starting function. This would mean that return stack is initialized as empty, and `RETF` exceptionally aborts when return stack is empty.
```

Comment:
```suggestion
Alternative logic for executing `RETF` in the top frame could be to exceptionally halt execution, because there is arguably no caller for the starting function. This would mean that the return stack is initialized as empty, and `RETF` exceptionally aborts when the return stack is empty.
```
(I'm pretty sure you didn't mean to remove that first part).